vars/pr.groovy

```diff
@@ -110,3 +110,37 @@ Map injectPRParams() {
     }
     return params
 }
+
+List<String> getAllLabelsFor(Map config = [:]) {
```

Comment:
Where do We use those functions? :)

Reply:
we will use them in vega core Jenkinsfile - possibly by adding `devnet` label, the pipeline will trigger Devnet CD for that PR.